draw-react-app/src/AnimatedTypography/AnimatedTypography.js

```diff
@@ -1,33 +1,118 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import "./CyberTruckWinner.css";
 import Box from "@mui/material/Box";
 import { Typography } from "@mui/material";
 
-  export const AnimatedMacbookWinner = ({ actualWinner }) => {
-    const elements = (
-    <Typography variant="h3" color="#30fcfc" className="a-text" key={actualWinner} style={{ padding: 10, display:'flex', whiteSpace:'nowrap', justifyContent:'flex-start', fontFamily: 'Kanit, sans-serif', fontWeight: 400 }}>
+export const AnimatedMacbookWinner = ({ actualWinner }) => {
+  const elements = (
+    <Typography
+      variant="h3"
+      color="#30fcfc"
+      key={actualWinner}
+      style={{
+        padding: 10,
+        display: "flex",
+        whiteSpace: "nowrap",
+        justifyContent: "flex-start",
+        fontFamily: "Kanit, sans-serif",
+        fontWeight: 400,
+      }}
+    >
       {actualWinner}
-    </Typography>);
-    const bar = <>
-    <Box position="absolute" className='animated-bar' height={20}></Box>
+    </Typography>
+  );
+  const bar = (
```

Comment:
```suggestion
  const bar = (
```
Shall we remove `bar`  and directly use the  element

Reply:
Fixed via [383235d](https://github.com/wso2/code-challenge-draw/pull/9/commits/383235d49b285b7bfc186557d0a98397c2be0d56)